en/docs/install.html

```diff
@@ -50,10 +50,18 @@
 </div>
 
 {% capture issueBody %}
-**Which operating system are you using:**
+<!-- *Before creating an issue please make sure you are using the latest version of yarn.* -->
 
+**Do you want to request a *feature* or report a *bug*?**
```

Comment:
Would someone actually use the links on the installation page to request a feature? I feel like it'd only be bug reports. I like the old format as it's pretty simple.